app/views/layouts/application.html.erb

```diff
@@ -1,15 +1,72 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Capter</title>
+   
+   <title>Members Only</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= favicon_link_tag asset_path('favicon-32x32.png') %>
+
   </head>
 
   <body>
-    <%= yield %>
+<% if flash[:notice] %>
+<div class="notification is-primary global-notification">
+<p class="notice"><%= notice %></p>
+</div>
+<% end %>
+<% if flash[:alert] %>
+<div class="notification is-danger global-notification">
+<p class="alert"><%= alert %></p>
+</div>
+<% end %>
```

Comment:
- [ ] put the if statements helper methods that you call here instead and the contents of the if statements in a partial that you render within the helper methods to remove the if/end logic from the view